[32mINFO    [0m integration.ha.test_backups:test_backups.py:490 Remove s3-credentials relation
[32mINFO    [0m juju.model:model.py:2971 Waiting for model:
  main/0 [idle] active: 
[32mINFO    [0m juju.model:model.py:2971 Waiting for model:
  main/0 [idle] active: 
[32mINFO    [0m juju.model:model.py:2971 Waiting for model:
  main/0 [idle] active: 
[32mINFO    [0m integration.ha.test_backups:test_backups.py:502 Re-add s3-credentials relation
[32mINFO    [0m juju.model:model.py:2971 Waiting for model:
  main/0 [idle] active: 
[32mINFO    [0m juju.model:model.py:2971 Waiting for model:
  main/0 [executing] maintenance: Configuring backup service...
[32mINFO    [0m juju.model:model.py:2971 Waiting for model:
  main/0 [idle] active: 
[32mINFO    [0m juju.model:model.py:2971 Waiting for model:
  main/0 [idle] active: 
[32mINFO    [0m integration.ha.test_backups:test_backups.py:511 Syncing credentials for microceph
[32mINFO    [0m juju.model:model.py:2971 Waiting for model:
  main/0 [idle] active: 
  s3-integrator/0 [idle] active: 
[32mINFO    [0m integration.helpers:helpers.py:347 Calling: GET -- https://10.138.71.161:9200/_recovery?human
[32mINFO    [0m integration.ha.helpers:helpers.py:247 Continuous writes result: namespace(max_stored_id=337, count=338, last_expected_id=337)
[32mINFO    [0m integration.helpers:helpers.py:347 Calling: GET -- https://10.138.71.161:9200/series_index/_search_shards
[32mINFO    [0m integration.helpers:helpers.py:347 Calling: POST -- https://10.138.71.161:9200/series_index/_refresh
[32mINFO    [0m integration.helpers:helpers.py:347 Calling: GET -- https://10.138.71.161:9200/series_index/_count
[32mINFO    [0m integration.helpers:helpers.py:347 Calling: GET -- https://10.138.71.161:9200/_recovery?human
[32mINFO    [0m integration.helpers:helpers.py:347 Calling: POST -- https://10.138.71.161:9200/series_index/_refresh
[32mINFO    [0m integration.helpers:helpers.py:347 Calling: GET -- https://10.138.71.161:9200/series_index/_count
[32mINFO    [0m integration.ha.helpers:helpers.py:591 Testing restore for series_index - original count pre-restore: 338, and now, new count: 308
[32mINFO    [0m integration.helpers:helpers.py:347 Calling: POST -- https://10.138.71.161:9200/series_index/_refresh
[32mINFO    [0m integration.helpers:helpers.py:347 Calling: GET -- https://10.138.71.161:9200/series_index/_count
[32mINFO    [0m integration.ha.conftest:conftest.py:39 



The writes have been cleared.




[32mINFO    [0m pytest_operator.plugin:plugin.py:862 Model status:

Model  Controller           Cloud/Region         Version  SLA          Timestamp
test   localhost-localhost  localhost/localhost  3.5.3    unsupported  02:05:59Z

App                       Version  Status  Scale  Charm                     Channel        Rev  Exposed  Message
failover                           active      2  opensearch                                 1  no       
main                               active      1  opensearch                                 2  no       
opensearch                         active      1  opensearch                                 0  no       
s3-integrator                      active      1  s3-integrator             latest/edge     41  no       
self-signed-certificates           active      1  self-signed-certificates  latest/stable  155  no       

Unit                         Workload  Agent  Machine  Public address  Ports     Message
failover/0                   active    idle   3        10.138.71.197   9200/tcp  
failover/1*                  active    idle   4        10.138.71.194   9200/tcp  
main/0*                      active    idle   5        10.138.71.161   9200/tcp  
opensearch/0*                active    idle   2        10.138.71.136   9200/tcp  
s3-integrator/0*             active    idle   1        10.138.71.88              
self-signed-certificates/0*  active    idle   0        10.138.71.131             

Machine  State    Address        Inst id        Base          AZ  Message
0        started  10.138.71.131  juju-82f0c1-0  ubuntu@22.04      Running
1        started  10.138.71.88   juju-82f0c1-1  ubuntu@22.04      Running
2        started  10.138.71.136  juju-82f0c1-2  ubuntu@22.04      Running
3        started  10.138.71.197  juju-82f0c1-3  ubuntu@22.04      Running
4        started  10.138.71.194  juju-82f0c1-4  ubuntu@22.04      Running
5        started  10.138.71.161  juju-82f0c1-5  ubuntu@22.04      Running

[32mINFO    [0m pytest_operator.plugin:plugin.py:868 Juju error logs:

machine-1: 01:51:39 ERROR juju.worker.dependency "kvm-container-provisioner" manifold worker returned unexpected error: container types not yet available
machine-1: 01:51:39 ERROR juju.worker.dependency "lxd-container-provisioner" manifold worker returned unexpected error: container types not yet available
unit-s3-integrator-0: 01:51:39 ERROR juju.worker.meterstatus error running "meter-status-changed": charm missing from disk
machine-0: 01:51:39 ERROR juju.worker.dependency "lxd-container-provisioner" manifold worker returned unexpected error: container types not yet available
machine-0: 01:51:39 ERROR juju.worker.dependency "kvm-container-provisioner" manifold worker returned unexpected error: container types not yet available
unit-self-signed-certificates-0: 01:51:39 ERROR juju.worker.meterstatus error running "meter-status-changed": charm missing from disk
machine-4: 01:51:55 ERROR juju.worker.dependency "lxd-container-provisioner" manifold worker returned unexpected error: container types not yet available
machine-4: 01:51:55 ERROR juju.worker.dependency "kvm-container-provisioner" manifold worker returned unexpected error: container types not yet available
unit-failover-1: 01:51:56 ERROR juju.worker.meterstatus error running "meter-status-changed": charm missing from disk
machine-5: 01:51:59 ERROR juju.worker.dependency "lxd-container-provisioner" manifold worker returned unexpected error: container types not yet available
machine-5: 01:51:59 ERROR juju.worker.dependency "kvm-container-provisioner" manifold worker returned unexpected error: container types not yet available
unit-main-0: 01:51:59 ERROR juju.worker.meterstatus error running "meter-status-changed": charm missing from disk
machine-2: 01:51:59 ERROR juju.worker.dependency "lxd-container-provisioner" manifold worker returned unexpected error: container types not yet available
machine-2: 01:51:59 ERROR juju.worker.dependency "kvm-container-provisioner" manifold worker returned unexpected error: container types not yet available
unit-opensearch-0: 01:51:59 ERROR juju.worker.meterstatus error running "meter-status-changed": charm missing from disk
machine-3: 01:52:01 ERROR juju.worker.dependency "lxd-container-provisioner" manifold worker returned unexpected error: container types not yet available
machine-3: 01:52:01 ERROR juju.worker.dependency "kvm-container-provisioner" manifold worker returned unexpected error: container types not yet available
unit-failover-0: 01:52:01 ERROR juju.worker.meterstatus error running "meter-status-changed": charm missing from disk
unit-main-0: 01:53:15 ERROR unit.main/0.juju-log [Errno 111] Connection refused
unit-main-0: 01:53:17 ERROR unit.main/0.juju-log upgrade-version-a:9: [Errno 111] Connection refused
unit-main-0: 01:53:18 ERROR unit.main/0.juju-log opensearch-peers:7: [Errno 111] Connection refused
unit-main-0: 01:53:18 ERROR unit.main/0.juju-log opensearch-peers:7: [Errno 111] Connection refused
unit-main-0: 01:53:18 ERROR unit.main/0.juju-log opensearch-peers:7: [Errno 111] Connection refused
unit-main-0: 01:53:18 ERROR unit.main/0.juju-log opensearch-peers:7: [Errno 111] Connection refused
unit-main-0: 01:53:19 ERROR unit.main/0.juju-log certificates:13: [Errno 111] Connection refused
unit-main-0: 01:53:23 ERROR unit.main/0.juju-log [Errno 111] Connection refused
unit-main-0: 01:53:24 ERROR unit.main/0.juju-log [Errno 111] Connection refused
unit-main-0: 01:53:25 ERROR unit.main/0.juju-log [Errno 111] Connection refused
unit-main-0: 01:53:26 ERROR unit.main/0.juju-log certificates:13: [Errno 111] Connection refused
unit-main-0: 01:53:26 ERROR unit.main/0.juju-log certificates:13: [Errno 111] Connection refused
unit-main-0: 01:53:26 ERROR unit.main/0.juju-log certificates:13: Cannot connect to the OpenSearch server...
unit-main-0: 01:53:26 ERROR unit.main/0.juju-log certificates:13: [Errno 111] Connection refused
unit-main-0: 01:53:26 ERROR unit.main/0.juju-log certificates:13: [Errno 111] Connection refused
unit-main-0: 01:53:26 ERROR unit.main/0.juju-log certificates:13: Cannot connect to the OpenSearch server...
unit-main-0: 01:53:26 ERROR unit.main/0.juju-log certificates:13: [Errno 111] Connection refused
unit-main-0: 01:53:26 ERROR unit.main/0.juju-log certificates:13: Cannot connect to the OpenSearch server...
unit-failover-1: 01:53:29 ERROR unit.failover/1.juju-log [Errno 111] Connection refused
unit-main-0: 01:53:29 ERROR unit.main/0.juju-log certificates:13: [Errno 111] Connection refused
unit-main-0: 01:53:30 ERROR unit.main/0.juju-log certificates:13: Cannot connect to the OpenSearch server...
unit-failover-1: 01:53:30 ERROR unit.failover/1.juju-log peer-cluster:10: [Errno 111] Connection refused
unit-failover-1: 01:53:30 ERROR unit.failover/1.juju-log peer-cluster:10: [Errno 111] Connection refused
unit-failover-1: 01:53:31 ERROR unit.failover/1.juju-log node-lock-fallback:2: [Errno 111] Connection refused
unit-failover-0: 01:53:32 ERROR unit.failover/0.juju-log [Errno 111] Connection refused
unit-failover-1: 01:53:32 ERROR unit.failover/1.juju-log node-lock-fallback:2: [Errno 111] Connection refused
unit-opensearch-0: 01:53:32 ERROR unit.opensearch/0.juju-log [Errno 111] Connection refused
unit-main-0: 01:53:33 ERROR unit.main/0.juju-log certificates:13: [Errno 111] Connection refused
unit-main-0: 01:53:33 ERROR unit.main/0.juju-log certificates:13: Cannot connect to the OpenSearch server...
unit-failover-0: 01:53:33 ERROR unit.failover/0.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-failover-0: 01:53:33 ERROR unit.failover/0.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-failover-1: 01:53:33 ERROR unit.failover/1.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-failover-1: 01:53:33 ERROR unit.failover/1.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-failover-1: 01:53:33 ERROR unit.failover/1.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-failover-1: 01:53:33 ERROR unit.failover/1.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-opensearch-0: 01:53:33 ERROR unit.opensearch/0.juju-log opensearch-peers:6: [Errno 111] Connection refused
unit-opensearch-0: 01:53:33 ERROR unit.opensearch/0.juju-log opensearch-peers:6: [Errno 111] Connection refused
unit-opensearch-0: 01:53:33 ERROR unit.opensearch/0.juju-log opensearch-peers:6: [Errno 111] Connection refused
unit-opensearch-0: 01:53:33 ERROR unit.opensearch/0.juju-log opensearch-peers:6: [Errno 111] Connection refused
unit-failover-0: 01:53:34 ERROR unit.failover/0.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-failover-1: 01:53:34 ERROR unit.failover/1.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-opensearch-0: 01:53:34 ERROR unit.opensearch/0.juju-log peer-cluster:11: [Errno 111] Connection refused
unit-failover-0: 01:53:34 ERROR unit.failover/0.juju-log peer-cluster-orchestrator:12: [Errno 111] Connection refused
unit-failover-1: 01:53:35 ERROR unit.failover/1.juju-log upgrade-version-a:3: [Errno 111] Connection refused
unit-opensearch-0: 01:53:35 ERROR unit.opensearch/0.juju-log peer-cluster:11: [Errno 111] Connection refused
unit-failover-0: 01:53:35 ERROR unit.failover/0.juju-log peer-cluster-orchestrator:12: [Errno 111] Connection refused
unit-main-0: 01:53:36 ERROR unit.main/0.juju-log certificates:13: [Errno 111] Connection refused
unit-main-0: 01:53:36 ERROR unit.main/0.juju-log certificates:13: Cannot connect to the OpenSearch server...
unit-failover-1: 01:53:36 ERROR unit.failover/1.juju-log certificates:14: [Errno 111] Connection refused
unit-opensearch-0: 01:53:36 ERROR unit.opensearch/0.juju-log certificates:15: [Errno 111] Connection refused
unit-failover-0: 01:53:36 ERROR unit.failover/0.juju-log upgrade-version-a:3: [Errno 111] Connection refused
unit-failover-1: 01:53:37 ERROR unit.failover/1.juju-log upgrade-version-a:3: [Errno 111] Connection refused
unit-opensearch-0: 01:53:37 ERROR unit.opensearch/0.juju-log opensearch-peers:6: [Errno 111] Connection refused
unit-opensearch-0: 01:53:37 ERROR unit.opensearch/0.juju-log opensearch-peers:6: [Errno 111] Connection refused
unit-opensearch-0: 01:53:37 ERROR unit.opensearch/0.juju-log opensearch-peers:6: [Errno 111] Connection refused
unit-opensearch-0: 01:53:37 ERROR unit.opensearch/0.juju-log opensearch-peers:6: [Errno 111] Connection refused
unit-failover-0: 01:53:37 ERROR unit.failover/0.juju-log certificates:14: [Errno 111] Connection refused
unit-failover-1: 01:53:38 ERROR unit.failover/1.juju-log certificates:14: [Errno 111] Connection refused
unit-opensearch-0: 01:53:38 ERROR unit.opensearch/0.juju-log peer-cluster:12: [Errno 111] Connection refused
unit-failover-0: 01:53:38 ERROR unit.failover/0.juju-log upgrade-version-a:3: [Errno 111] Connection refused
unit-main-0: 01:53:39 ERROR unit.main/0.juju-log certificates:13: [Errno 111] Connection refused
unit-main-0: 01:53:39 ERROR unit.main/0.juju-log certificates:13: Cannot connect to the OpenSearch server...
unit-failover-1: 01:53:39 ERROR unit.failover/1.juju-log certificates:14: [Errno 111] Connection refused
unit-failover-0: 01:53:39 ERROR unit.failover/0.juju-log certificates:14: [Errno 111] Connection refused
unit-opensearch-0: 01:53:39 ERROR unit.opensearch/0.juju-log certificates:15: [Errno 111] Connection refused
unit-failover-1: 01:53:39 ERROR unit.failover/1.juju-log upgrade-version-a:3: [Errno 111] Connection refused
unit-opensearch-0: 01:53:40 ERROR unit.opensearch/0.juju-log peer-cluster:12: [Errno 111] Connection refused
unit-failover-0: 01:53:40 ERROR unit.failover/0.juju-log upgrade-version-a:3: [Errno 111] Connection refused
unit-failover-1: 01:53:40 ERROR unit.failover/1.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-failover-1: 01:53:40 ERROR unit.failover/1.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-opensearch-0: 01:53:40 ERROR unit.opensearch/0.juju-log certificates:15: [Errno 111] Connection refused
unit-failover-1: 01:53:40 ERROR unit.failover/1.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-failover-0: 01:53:40 ERROR unit.failover/0.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-failover-1: 01:53:40 ERROR unit.failover/1.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-failover-1: 01:53:40 ERROR unit.failover/1.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-failover-1: 01:53:40 ERROR unit.failover/1.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-failover-1: 01:53:40 ERROR unit.failover/1.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-failover-1: 01:53:41 ERROR unit.failover/1.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-failover-1: 01:53:41 ERROR unit.failover/1.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-failover-0: 01:53:41 ERROR unit.failover/0.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-failover-0: 01:53:41 ERROR unit.failover/0.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-failover-0: 01:53:41 ERROR unit.failover/0.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-failover-0: 01:53:41 ERROR unit.failover/0.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-failover-0: 01:53:41 ERROR unit.failover/0.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-failover-0: 01:53:41 ERROR unit.failover/0.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-opensearch-0: 01:53:41 ERROR unit.opensearch/0.juju-log peer-cluster:12: [Errno 111] Connection refused
unit-failover-1: 01:53:41 ERROR unit.failover/1.juju-log peer-cluster-orchestrator:12: [Errno 111] Connection refused
unit-failover-0: 01:53:41 ERROR unit.failover/0.juju-log peer-cluster-orchestrator:12: [Errno 111] Connection refused
unit-opensearch-0: 01:53:42 ERROR unit.opensearch/0.juju-log peer-cluster:12: [Errno 111] Connection refused
unit-failover-1: 01:53:42 ERROR unit.failover/1.juju-log peer-cluster-orchestrator:12: [Errno 111] Connection refused
unit-failover-0: 01:53:42 ERROR unit.failover/0.juju-log certificates:14: [Errno 111] Connection refused
unit-opensearch-0: 01:53:43 ERROR unit.opensearch/0.juju-log upgrade-version-a:5: [Errno 111] Connection refused
unit-failover-1: 01:53:43 ERROR unit.failover/1.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-failover-0: 01:53:43 ERROR unit.failover/0.juju-log node-lock-fallback:2: [Errno 111] Connection refused
unit-failover-1: 01:53:43 ERROR unit.failover/1.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-failover-1: 01:53:43 ERROR unit.failover/1.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-failover-1: 01:53:43 ERROR unit.failover/1.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-failover-1: 01:53:43 ERROR unit.failover/1.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-opensearch-0: 01:53:44 ERROR unit.opensearch/0.juju-log peer-cluster:12: [Errno 111] Connection refused
unit-failover-0: 01:53:44 ERROR unit.failover/0.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-failover-0: 01:53:44 ERROR unit.failover/0.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-failover-0: 01:53:44 ERROR unit.failover/0.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-failover-1: 01:53:44 ERROR unit.failover/1.juju-log peer-cluster-orchestrator:12: [Errno 111] Connection refused
unit-opensearch-0: 01:53:45 ERROR unit.opensearch/0.juju-log opensearch-peers:6: [Errno 111] Connection refused
unit-failover-0: 01:53:45 ERROR unit.failover/0.juju-log node-lock-fallback:2: [Errno 111] Connection refused
unit-opensearch-0: 01:53:45 ERROR unit.opensearch/0.juju-log opensearch-peers:6: [Errno 111] Connection refused
unit-opensearch-0: 01:53:45 ERROR unit.opensearch/0.juju-log opensearch-peers:6: [Errno 111] Connection refused
unit-opensearch-0: 01:53:45 ERROR unit.opensearch/0.juju-log opensearch-peers:6: [Errno 111] Connection refused
unit-opensearch-0: 01:53:46 ERROR unit.opensearch/0.juju-log peer-cluster:12: [Errno 111] Connection refused
unit-failover-0: 01:53:46 ERROR unit.failover/0.juju-log peer-cluster:10: [Errno 111] Connection refused
unit-failover-0: 01:53:47 ERROR unit.failover/0.juju-log peer-cluster:10: [Errno 111] Connection refused
unit-failover-1: 01:54:04 ERROR unit.failover/1.juju-log peer-cluster:10: [Errno 111] Connection refused
unit-failover-0: 01:54:04 ERROR unit.failover/0.juju-log peer-cluster:10: [Errno 111] Connection refused
unit-failover-1: 01:54:04 ERROR unit.failover/1.juju-log peer-cluster:10: [Errno 111] Connection refused
unit-failover-1: 01:54:04 ERROR unit.failover/1.juju-log peer-cluster:10: [Errno 111] Connection refused
unit-failover-1: 01:54:05 ERROR unit.failover/1.juju-log peer-cluster:10: [Errno 111] Connection refused
unit-failover-1: 01:54:05 ERROR unit.failover/1.juju-log peer-cluster:10: [Errno 111] Connection refused
unit-failover-1: 01:54:05 ERROR unit.failover/1.juju-log peer-cluster:10: [Errno 111] Connection refused
unit-failover-1: 01:54:05 ERROR unit.failover/1.juju-log peer-cluster:10: [Errno 111] Connection refused
unit-failover-1: 01:54:12 ERROR unit.failover/1.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-failover-1: 01:54:12 ERROR unit.failover/1.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-failover-0: 01:54:12 ERROR unit.failover/0.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-failover-0: 01:54:12 ERROR unit.failover/0.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-failover-1: 01:54:12 ERROR unit.failover/1.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-failover-1: 01:54:12 ERROR unit.failover/1.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-failover-0: 01:54:12 ERROR unit.failover/0.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-failover-1: 01:54:12 ERROR unit.failover/1.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-failover-0: 01:54:12 ERROR unit.failover/0.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-failover-1: 01:54:12 ERROR unit.failover/1.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-failover-1: 01:54:12 ERROR unit.failover/1.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-failover-0: 01:54:13 ERROR unit.failover/0.juju-log peer-cluster:10: [Errno 111] Connection refused
unit-failover-0: 01:54:13 ERROR unit.failover/0.juju-log peer-cluster:10: [Errno 111] Connection refused
unit-failover-1: 01:54:13 ERROR unit.failover/1.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-opensearch-0: 01:54:13 ERROR unit.opensearch/0.juju-log peer-cluster:12: [Errno 111] Connection refused
unit-failover-1: 01:54:13 ERROR unit.failover/1.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-failover-1: 01:54:14 ERROR unit.failover/1.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-failover-1: 01:54:14 ERROR unit.failover/1.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-failover-1: 01:54:14 ERROR unit.failover/1.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-failover-1: 01:54:14 ERROR unit.failover/1.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-failover-1: 01:54:14 ERROR unit.failover/1.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-failover-1: 01:54:14 ERROR unit.failover/1.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-failover-1: 01:54:14 ERROR unit.failover/1.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-failover-1: 01:54:14 ERROR unit.failover/1.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-failover-1: 01:54:14 ERROR unit.failover/1.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-failover-0: 01:54:14 ERROR unit.failover/0.juju-log certificates:14: [Errno 111] Connection refused
unit-opensearch-0: 01:54:14 ERROR unit.opensearch/0.juju-log peer-cluster:11: [Errno 111] Connection refused
unit-failover-0: 01:54:14 ERROR unit.failover/0.juju-log certificates:14: [Errno 111] Connection refused
unit-opensearch-0: 01:54:15 ERROR unit.opensearch/0.juju-log peer-cluster:11: [Errno 111] Connection refused
unit-failover-1: 01:54:15 ERROR unit.failover/1.juju-log certificates:14: [Errno 111] Connection refused
unit-failover-1: 01:54:15 ERROR unit.failover/1.juju-log certificates:14: [Errno 111] Connection refused
unit-opensearch-0: 01:54:15 ERROR unit.opensearch/0.juju-log peer-cluster:11: [Errno 111] Connection refused
unit-opensearch-0: 01:54:16 ERROR unit.opensearch/0.juju-log peer-cluster:11: [Errno 111] Connection refused
unit-opensearch-0: 01:54:17 ERROR unit.opensearch/0.juju-log opensearch-peers:6: [Errno 111] Connection refused
unit-failover-0: 01:54:18 ERROR unit.failover/0.juju-log [Errno 111] Connection refused
unit-failover-0: 01:54:18 ERROR unit.failover/0.juju-log [Errno 111] Connection refused
unit-opensearch-0: 01:54:18 ERROR unit.opensearch/0.juju-log opensearch-peers:6: [Errno 111] Connection refused
unit-opensearch-0: 01:54:18 ERROR unit.opensearch/0.juju-log opensearch-peers:6: [Errno 111] Connection refused
unit-opensearch-0: 01:54:18 ERROR unit.opensearch/0.juju-log opensearch-peers:6: [Errno 111] Connection refused
unit-failover-1: 01:54:18 ERROR unit.failover/1.juju-log [Errno 111] Connection refused
unit-failover-1: 01:54:18 ERROR unit.failover/1.juju-log [Errno 111] Connection refused
unit-failover-0: 01:54:19 ERROR unit.failover/0.juju-log [Errno 111] Connection refused
unit-failover-0: 01:54:19 ERROR unit.failover/0.juju-log [Errno 111] Connection refused
unit-opensearch-0: 01:54:19 ERROR unit.opensearch/0.juju-log opensearch-peers:6: [Errno 111] Connection refused
unit-opensearch-0: 01:54:19 ERROR unit.opensearch/0.juju-log opensearch-peers:6: [Errno 111] Connection refused
unit-opensearch-0: 01:54:19 ERROR unit.opensearch/0.juju-log opensearch-peers:6: [Errno 111] Connection refused
unit-opensearch-0: 01:54:19 ERROR unit.opensearch/0.juju-log opensearch-peers:6: [Errno 111] Connection refused
unit-failover-1: 01:54:20 ERROR unit.failover/1.juju-log [Errno 111] Connection refused
unit-failover-1: 01:54:20 ERROR unit.failover/1.juju-log [Errno 111] Connection refused
unit-failover-0: 01:54:20 ERROR unit.failover/0.juju-log certificates:14: [Errno 111] Connection refused
unit-failover-0: 01:54:20 ERROR unit.failover/0.juju-log certificates:14: [Errno 111] Connection refused
unit-failover-0: 01:54:20 ERROR unit.failover/0.juju-log certificates:14: [Errno 111] Connection refused
unit-failover-0: 01:54:20 ERROR unit.failover/0.juju-log certificates:14: Cannot connect to the OpenSearch server...
unit-failover-0: 01:54:20 ERROR unit.failover/0.juju-log certificates:14: [Errno 111] Connection refused
unit-failover-0: 01:54:20 ERROR unit.failover/0.juju-log certificates:14: [Errno 111] Connection refused
unit-opensearch-0: 01:54:20 ERROR unit.opensearch/0.juju-log certificates:15: [Errno 111] Connection refused
unit-failover-0: 01:54:20 ERROR unit.failover/0.juju-log certificates:14: [Errno 111] Connection refused
unit-failover-0: 01:54:21 ERROR unit.failover/0.juju-log certificates:14: [Errno 111] Connection refused
unit-failover-0: 01:54:21 ERROR unit.failover/0.juju-log certificates:14: [Errno 111] Connection refused
unit-failover-1: 01:54:21 ERROR unit.failover/1.juju-log [Errno 111] Connection refused
unit-failover-0: 01:54:21 ERROR unit.failover/0.juju-log certificates:14: [Errno 111] Connection refused
unit-failover-1: 01:54:21 ERROR unit.failover/1.juju-log [Errno 111] Connection refused
unit-failover-0: 01:54:21 ERROR unit.failover/0.juju-log certificates:14: [Errno 111] Connection refused
unit-failover-0: 01:54:21 ERROR unit.failover/0.juju-log certificates:14: [Errno 111] Connection refused
unit-failover-0: 01:54:21 ERROR unit.failover/0.juju-log certificates:14: [Errno 111] Connection refused
unit-failover-1: 01:54:21 ERROR unit.failover/1.juju-log [Errno 111] Connection refused
unit-failover-1: 01:54:21 ERROR unit.failover/1.juju-log Cannot connect to the OpenSearch server...
unit-failover-1: 01:54:21 ERROR unit.failover/1.juju-log [Errno 111] Connection refused
unit-failover-1: 01:54:21 ERROR unit.failover/1.juju-log [Errno 111] Connection refused
unit-failover-0: 01:54:21 ERROR unit.failover/0.juju-log certificates:14: [Errno 111] Connection refused
unit-failover-1: 01:54:21 ERROR unit.failover/1.juju-log [Errno 111] Connection refused
unit-failover-1: 01:54:21 ERROR unit.failover/1.juju-log [Errno 111] Connection refused
unit-failover-0: 01:54:21 ERROR unit.failover/0.juju-log certificates:14: [Errno 111] Connection refused
unit-failover-0: 01:54:22 ERROR unit.failover/0.juju-log certificates:14: [Errno 111] Connection refused
unit-failover-0: 01:54:22 ERROR unit.failover/0.juju-log certificates:14: Cannot connect to the OpenSearch server...
unit-failover-0: 01:54:22 ERROR unit.failover/0.juju-log certificates:14: [Errno 111] Connection refused
unit-failover-1: 01:54:22 ERROR unit.failover/1.juju-log [Errno 111] Connection refused
unit-failover-1: 01:54:22 ERROR unit.failover/1.juju-log Cannot connect to the OpenSearch server...
unit-failover-1: 01:54:22 ERROR unit.failover/1.juju-log [Errno 111] Connection refused
unit-failover-1: 01:54:22 ERROR unit.failover/1.juju-log [Errno 111] Connection refused
unit-failover-1: 01:54:23 ERROR unit.failover/1.juju-log [Errno 111] Connection refused
unit-failover-1: 01:54:23 ERROR unit.failover/1.juju-log [Errno 111] Connection refused
unit-failover-1: 01:54:24 ERROR unit.failover/1.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-failover-1: 01:54:24 ERROR unit.failover/1.juju-log opensearch-peers:1: Cannot connect to the OpenSearch server...
unit-failover-1: 01:54:24 ERROR unit.failover/1.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-failover-1: 01:54:24 ERROR unit.failover/1.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-opensearch-0: 01:54:24 ERROR unit.opensearch/0.juju-log [Errno 111] Connection refused
unit-failover-1: 01:54:24 ERROR unit.failover/1.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-failover-1: 01:54:24 ERROR unit.failover/1.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-failover-1: 01:54:24 ERROR unit.failover/1.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-failover-1: 01:54:24 ERROR unit.failover/1.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-failover-1: 01:54:24 ERROR unit.failover/1.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-failover-1: 01:54:24 ERROR unit.failover/1.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-failover-1: 01:54:24 ERROR unit.failover/1.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-failover-1: 01:54:24 ERROR unit.failover/1.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-failover-1: 01:54:24 ERROR unit.failover/1.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-failover-1: 01:54:25 ERROR unit.failover/1.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-failover-1: 01:54:25 ERROR unit.failover/1.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-failover-1: 01:54:25 ERROR unit.failover/1.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-opensearch-0: 01:54:25 ERROR unit.opensearch/0.juju-log [Errno 111] Connection refused
unit-failover-1: 01:54:25 ERROR unit.failover/1.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-failover-0: 01:54:25 ERROR unit.failover/0.juju-log certificates:14: [Errno 111] Connection refused
unit-failover-1: 01:54:26 ERROR unit.failover/1.juju-log certificates:14: [Errno 111] Connection refused
unit-failover-1: 01:54:26 ERROR unit.failover/1.juju-log certificates:14: Cannot connect to the OpenSearch server...
unit-failover-1: 01:54:26 ERROR unit.failover/1.juju-log certificates:14: [Errno 111] Connection refused
unit-opensearch-0: 01:54:26 ERROR unit.opensearch/0.juju-log [Errno 111] Connection refused
unit-failover-1: 01:54:26 ERROR unit.failover/1.juju-log certificates:14: [Errno 111] Connection refused
unit-failover-1: 01:54:26 ERROR unit.failover/1.juju-log certificates:14: [Errno 111] Connection refused
unit-failover-1: 01:54:26 ERROR unit.failover/1.juju-log certificates:14: [Errno 111] Connection refused
unit-opensearch-0: 01:54:26 ERROR unit.opensearch/0.juju-log [Errno 111] Connection refused
unit-opensearch-0: 01:54:26 ERROR unit.opensearch/0.juju-log Cannot connect to the OpenSearch server...
unit-opensearch-0: 01:54:26 ERROR unit.opensearch/0.juju-log [Errno 111] Connection refused
unit-opensearch-0: 01:54:26 ERROR unit.opensearch/0.juju-log [Errno 111] Connection refused
unit-opensearch-0: 01:54:27 ERROR unit.opensearch/0.juju-log [Errno 111] Connection refused
unit-opensearch-0: 01:54:27 ERROR unit.opensearch/0.juju-log Cannot connect to the OpenSearch server...
unit-opensearch-0: 01:54:27 ERROR unit.opensearch/0.juju-log [Errno 111] Connection refused
unit-failover-1: 01:54:28 ERROR unit.failover/1.juju-log [Errno 111] Connection refused
unit-failover-1: 01:54:28 ERROR unit.failover/1.juju-log Cannot connect to the OpenSearch server...
unit-failover-1: 01:54:28 ERROR unit.failover/1.juju-log [Errno 111] Connection refused
unit-opensearch-0: 01:54:28 ERROR unit.opensearch/0.juju-log [Errno 111] Connection refused
unit-failover-1: 01:54:28 ERROR unit.failover/1.juju-log [Errno 111] Connection refused
unit-failover-1: 01:54:28 ERROR unit.failover/1.juju-log [Errno 111] Connection refused
unit-failover-1: 01:54:28 ERROR unit.failover/1.juju-log [Errno 111] Connection refused
unit-failover-0: 01:54:28 ERROR unit.failover/0.juju-log certificates:14: [Errno 111] Connection refused
unit-opensearch-0: 01:54:28 ERROR unit.opensearch/0.juju-log peer-cluster:12: [Errno 111] Connection refused
unit-opensearch-0: 01:54:28 ERROR unit.opensearch/0.juju-log peer-cluster:12: Cannot connect to the OpenSearch server...
unit-opensearch-0: 01:54:29 ERROR unit.opensearch/0.juju-log peer-cluster:12: [Errno 111] Connection refused
unit-opensearch-0: 01:54:29 ERROR unit.opensearch/0.juju-log peer-cluster:12: [Errno 111] Connection refused
unit-failover-1: 01:54:29 ERROR unit.failover/1.juju-log [Errno 111] Connection refused
unit-failover-1: 01:54:29 ERROR unit.failover/1.juju-log Cannot connect to the OpenSearch server...
unit-failover-1: 01:54:29 ERROR unit.failover/1.juju-log [Errno 111] Connection refused
unit-failover-1: 01:54:29 ERROR unit.failover/1.juju-log [Errno 111] Connection refused
unit-failover-1: 01:54:29 ERROR unit.failover/1.juju-log [Errno 111] Connection refused
unit-failover-1: 01:54:29 ERROR unit.failover/1.juju-log [Errno 111] Connection refused
unit-opensearch-0: 01:54:30 ERROR unit.opensearch/0.juju-log opensearch-peers:6: [Errno 111] Connection refused
unit-opensearch-0: 01:54:30 ERROR unit.opensearch/0.juju-log opensearch-peers:6: Cannot connect to the OpenSearch server...
unit-opensearch-0: 01:54:30 ERROR unit.opensearch/0.juju-log opensearch-peers:6: [Errno 111] Connection refused
unit-failover-1: 01:54:30 ERROR unit.failover/1.juju-log [Errno 111] Connection refused
unit-failover-1: 01:54:30 ERROR unit.failover/1.juju-log Cannot connect to the OpenSearch server...
unit-failover-1: 01:54:30 ERROR unit.failover/1.juju-log [Errno 111] Connection refused
unit-opensearch-0: 01:54:30 ERROR unit.opensearch/0.juju-log opensearch-peers:6: [Errno 111] Connection refused
unit-failover-1: 01:54:30 ERROR unit.failover/1.juju-log [Errno 111] Connection refused
unit-opensearch-0: 01:54:30 ERROR unit.opensearch/0.juju-log opensearch-peers:6: [Errno 111] Connection refused
unit-opensearch-0: 01:54:30 ERROR unit.opensearch/0.juju-log opensearch-peers:6: [Errno 111] Connection refused
unit-failover-1: 01:54:30 ERROR unit.failover/1.juju-log [Errno 111] Connection refused
unit-failover-1: 01:54:30 ERROR unit.failover/1.juju-log [Errno 111] Connection refused
unit-opensearch-0: 01:54:30 ERROR unit.opensearch/0.juju-log opensearch-peers:6: [Errno 111] Connection refused
unit-failover-0: 01:54:31 ERROR unit.failover/0.juju-log certificates:14: [Errno 111] Connection refused
unit-failover-1: 01:54:31 ERROR unit.failover/1.juju-log [Errno 111] Connection refused
unit-failover-1: 01:54:31 ERROR unit.failover/1.juju-log Cannot connect to the OpenSearch server...
unit-failover-1: 01:54:31 ERROR unit.failover/1.juju-log [Errno 111] Connection refused
unit-failover-1: 01:54:31 ERROR unit.failover/1.juju-log [Errno 111] Connection refused
unit-failover-1: 01:54:31 ERROR unit.failover/1.juju-log [Errno 111] Connection refused
unit-failover-1: 01:54:31 ERROR unit.failover/1.juju-log [Errno 111] Connection refused
unit-failover-1: 01:54:32 ERROR unit.failover/1.juju-log peer-cluster:10: [Errno 111] Connection refused
unit-failover-1: 01:54:32 ERROR unit.failover/1.juju-log peer-cluster:10: Cannot connect to the OpenSearch server...
unit-failover-1: 01:54:32 ERROR unit.failover/1.juju-log peer-cluster:10: [Errno 111] Connection refused
unit-failover-1: 01:54:32 ERROR unit.failover/1.juju-log peer-cluster:10: [Errno 111] Connection refused
unit-failover-1: 01:54:33 ERROR unit.failover/1.juju-log peer-cluster:10: [Errno 111] Connection refused
unit-failover-1: 01:54:33 ERROR unit.failover/1.juju-log peer-cluster:10: [Errno 111] Connection refused
unit-failover-1: 01:54:33 ERROR unit.failover/1.juju-log peer-cluster:10: [Errno 111] Connection refused
unit-failover-1: 01:54:33 ERROR unit.failover/1.juju-log peer-cluster:10: [Errno 111] Connection refused
unit-failover-1: 01:54:33 ERROR unit.failover/1.juju-log peer-cluster:10: [Errno 111] Connection refused
unit-failover-1: 01:54:34 ERROR unit.failover/1.juju-log peer-cluster:10: [Errno 111] Connection refused
unit-failover-0: 01:54:34 ERROR unit.failover/0.juju-log certificates:14: [Errno 111] Connection refused
unit-failover-1: 01:54:34 ERROR unit.failover/1.juju-log peer-cluster:10: [Errno 111] Connection refused
unit-failover-1: 01:54:34 ERROR unit.failover/1.juju-log peer-cluster:10: [Errno 111] Connection refused
unit-failover-1: 01:54:35 ERROR unit.failover/1.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-failover-1: 01:54:35 ERROR unit.failover/1.juju-log opensearch-peers:1: Cannot connect to the OpenSearch server...
unit-failover-1: 01:54:35 ERROR unit.failover/1.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-failover-1: 01:54:36 ERROR unit.failover/1.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-failover-1: 01:54:36 ERROR unit.failover/1.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-failover-1: 01:54:36 ERROR unit.failover/1.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-failover-1: 01:54:36 ERROR unit.failover/1.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-failover-1: 01:54:36 ERROR unit.failover/1.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-failover-1: 01:54:38 ERROR unit.failover/1.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-failover-1: 01:54:38 ERROR unit.failover/1.juju-log opensearch-peers:1: Cannot connect to the OpenSearch server...
unit-opensearch-0: 01:54:38 ERROR unit.opensearch/0.juju-log peer-cluster:12: [Errno 111] Connection refused
unit-failover-1: 01:54:38 ERROR unit.failover/1.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-opensearch-0: 01:54:38 ERROR unit.opensearch/0.juju-log peer-cluster:12: Cannot connect to the OpenSearch server...
unit-opensearch-0: 01:54:38 ERROR unit.opensearch/0.juju-log peer-cluster:12: [Errno 111] Connection refused
unit-opensearch-0: 01:54:38 ERROR unit.opensearch/0.juju-log peer-cluster:12: [Errno 111] Connection refused
unit-failover-1: 01:54:38 ERROR unit.failover/1.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-failover-1: 01:54:38 ERROR unit.failover/1.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-failover-1: 01:54:38 ERROR unit.failover/1.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-failover-1: 01:54:39 ERROR unit.failover/1.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-failover-1: 01:54:39 ERROR unit.failover/1.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-failover-0: 01:54:39 ERROR unit.failover/0.juju-log [Errno 111] Connection refused
unit-failover-0: 01:54:39 ERROR unit.failover/0.juju-log [Errno 111] Connection refused
unit-failover-0: 01:54:39 ERROR unit.failover/0.juju-log [Errno 111] Connection refused
unit-failover-0: 01:54:40 ERROR unit.failover/0.juju-log [Errno 111] Connection refused
unit-failover-0: 01:54:40 ERROR unit.failover/0.juju-log [Errno 111] Connection refused
unit-failover-1: 01:54:40 ERROR unit.failover/1.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-failover-1: 01:54:40 ERROR unit.failover/1.juju-log opensearch-peers:1: Cannot connect to the OpenSearch server...
unit-failover-1: 01:54:40 ERROR unit.failover/1.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-opensearch-0: 01:54:40 ERROR unit.opensearch/0.juju-log peer-cluster:12: [Errno 111] Connection refused
unit-opensearch-0: 01:54:40 ERROR unit.opensearch/0.juju-log peer-cluster:12: Cannot connect to the OpenSearch server...
unit-opensearch-0: 01:54:40 ERROR unit.opensearch/0.juju-log peer-cluster:12: [Errno 111] Connection refused
unit-opensearch-0: 01:54:40 ERROR unit.opensearch/0.juju-log peer-cluster:12: [Errno 111] Connection refused
unit-failover-1: 01:54:40 ERROR unit.failover/1.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-opensearch-0: 01:54:40 ERROR unit.opensearch/0.juju-log peer-cluster:12: [Errno 111] Connection refused
unit-opensearch-0: 01:54:40 ERROR unit.opensearch/0.juju-log peer-cluster:12: [Errno 111] Connection refused
unit-failover-1: 01:54:41 ERROR unit.failover/1.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-opensearch-0: 01:54:41 ERROR unit.opensearch/0.juju-log peer-cluster:12: [Errno 111] Connection refused
unit-failover-1: 01:54:41 ERROR unit.failover/1.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-failover-1: 01:54:41 ERROR unit.failover/1.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-failover-1: 01:54:41 ERROR unit.failover/1.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-opensearch-0: 01:54:41 ERROR unit.opensearch/0.juju-log peer-cluster:12: [Errno 111] Connection refused
unit-failover-1: 01:54:41 ERROR unit.failover/1.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-failover-1: 01:54:41 ERROR unit.failover/1.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-failover-1: 01:54:41 ERROR unit.failover/1.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-opensearch-0: 01:54:41 ERROR unit.opensearch/0.juju-log peer-cluster:12: [Errno 111] Connection refused
unit-opensearch-0: 01:54:41 ERROR unit.opensearch/0.juju-log peer-cluster:12: Cannot connect to the OpenSearch server...
unit-failover-1: 01:54:42 ERROR unit.failover/1.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-opensearch-0: 01:54:42 ERROR unit.opensearch/0.juju-log peer-cluster:12: [Errno 111] Connection refused
unit-failover-1: 01:54:42 ERROR unit.failover/1.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-failover-1: 01:54:42 ERROR unit.failover/1.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-failover-1: 01:54:42 ERROR unit.failover/1.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-failover-1: 01:54:43 ERROR unit.failover/1.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-failover-1: 01:54:43 ERROR unit.failover/1.juju-log opensearch-peers:1: Cannot connect to the OpenSearch server...
unit-failover-1: 01:54:43 ERROR unit.failover/1.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-failover-1: 01:54:44 ERROR unit.failover/1.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-failover-1: 01:54:44 ERROR unit.failover/1.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-failover-1: 01:54:44 ERROR unit.failover/1.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-failover-1: 01:54:44 ERROR unit.failover/1.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-opensearch-0: 01:54:45 ERROR unit.opensearch/0.juju-log peer-cluster:12: [Errno 111] Connection refused
unit-failover-1: 01:54:45 ERROR unit.failover/1.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-failover-0: 01:54:45 ERROR unit.failover/0.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-failover-1: 01:54:46 ERROR unit.failover/1.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-failover-1: 01:54:46 ERROR unit.failover/1.juju-log opensearch-peers:1: Cannot connect to the OpenSearch server...
unit-failover-1: 01:54:46 ERROR unit.failover/1.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-failover-1: 01:54:46 ERROR unit.failover/1.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-failover-1: 01:54:46 ERROR unit.failover/1.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-failover-1: 01:54:46 ERROR unit.failover/1.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-failover-0: 01:54:47 ERROR unit.failover/0.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-failover-0: 01:54:47 ERROR unit.failover/0.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-failover-1: 01:54:47 ERROR unit.failover/1.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-failover-0: 01:54:47 ERROR unit.failover/0.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-failover-1: 01:54:47 ERROR unit.failover/1.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-failover-1: 01:54:47 ERROR unit.failover/1.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-opensearch-0: 01:54:48 ERROR unit.opensearch/0.juju-log peer-cluster:12: [Errno 111] Connection refused
unit-failover-1: 01:54:48 ERROR unit.failover/1.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-failover-1: 01:54:49 ERROR unit.failover/1.juju-log upgrade-version-a:3: [Errno 111] Connection refused
unit-failover-1: 01:54:49 ERROR unit.failover/1.juju-log upgrade-version-a:3: Cannot connect to the OpenSearch server...
unit-failover-1: 01:54:49 ERROR unit.failover/1.juju-log upgrade-version-a:3: [Errno 111] Connection refused
unit-failover-1: 01:54:49 ERROR unit.failover/1.juju-log upgrade-version-a:3: [Errno 111] Connection refused
unit-opensearch-0: 01:54:51 ERROR unit.opensearch/0.juju-log peer-cluster:12: [Errno 111] Connection refused
unit-opensearch-0: 01:54:54 ERROR unit.opensearch/0.juju-log peer-cluster:12: [Errno 111] Connection refused
unit-failover-1: 01:56:47 ERROR unit.failover/1.juju-log [Errno 111] Connection refused
unit-failover-1: 01:56:47 ERROR unit.failover/1.juju-log Cannot connect to the OpenSearch server...
unit-failover-1: 01:56:47 ERROR unit.failover/1.juju-log [Errno 111] Connection refused
unit-failover-1: 01:56:47 ERROR unit.failover/1.juju-log [Errno 111] Connection refused
unit-failover-1: 01:56:47 ERROR unit.failover/1.juju-log [Errno 111] Connection refused
unit-failover-1: 01:56:47 ERROR unit.failover/1.juju-log [Errno 111] Connection refused
unit-failover-1: 01:56:48 ERROR unit.failover/1.juju-log [Errno 111] Connection refused
unit-failover-1: 01:56:48 ERROR unit.failover/1.juju-log [Errno 111] Connection refused
unit-failover-1: 01:56:48 ERROR unit.failover/1.juju-log [Errno 111] Connection refused
unit-failover-1: 01:56:49 ERROR unit.failover/1.juju-log [Errno 111] Connection refused
unit-failover-1: 01:56:49 ERROR unit.failover/1.juju-log Cannot connect to the OpenSearch server...
unit-failover-1: 01:56:49 ERROR unit.failover/1.juju-log [Errno 111] Connection refused
unit-failover-1: 01:56:52 ERROR unit.failover/1.juju-log [Errno 111] Connection refused
unit-failover-1: 01:56:55 ERROR unit.failover/1.juju-log [Errno 111] Connection refused
unit-failover-1: 01:56:58 ERROR unit.failover/1.juju-log [Errno 111] Connection refused
unit-failover-1: 01:57:01 ERROR unit.failover/1.juju-log [Errno 111] Connection refused
unit-main-0: 01:58:34 ERROR unit.main/0.juju-log s3-credentials:16: Plugin repository-s3 missing: ['access-key', 'secret-key']
unit-main-0: 01:58:36 ERROR unit.main/0.juju-log s3-credentials:16: Plugin repository-s3 missing: ['access-key', 'secret-key']
unit-main-0: 01:58:38 ERROR unit.main/0.juju-log s3-credentials:16: Plugin repository-s3 missing: ['access-key', 'secret-key']
unit-main-0: 01:58:46 ERROR unit.main/0.juju-log s3-credentials:16: Request PUT to https://10.138.71.161:9200/_snapshot/s3-repository with payload: {'type': 's3', 'settings': {'endpoint': 'http://localhost', 'protocol': 'http', 'bucket': 'error', 'base_path': '/', 'region': 'default'}} failed.(Attempts left: 5)
	Error: 500 Server Error: Internal Server Error for url: https://10.138.71.161:9200/_snapshot/s3-repository
unit-main-0: 01:58:47 ERROR unit.main/0.juju-log s3-credentials:16: Request PUT to https://10.138.71.161:9200/_snapshot/s3-repository with payload: {'type': 's3', 'settings': {'endpoint': 'http://localhost', 'protocol': 'http', 'bucket': 'error', 'base_path': '/', 'region': 'default'}} failed.(Attempts left: 4)
	Error: 500 Server Error: Internal Server Error for url: https://10.138.71.161:9200/_snapshot/s3-repository
unit-main-0: 01:58:49 ERROR unit.main/0.juju-log s3-credentials:16: Request PUT to https://10.138.71.161:9200/_snapshot/s3-repository with payload: {'type': 's3', 'settings': {'endpoint': 'http://localhost', 'protocol': 'http', 'bucket': 'error', 'base_path': '/', 'region': 'default'}} failed.(Attempts left: 3)
	Error: 500 Server Error: Internal Server Error for url: https://10.138.71.161:9200/_snapshot/s3-repository
unit-main-0: 01:58:50 ERROR unit.main/0.juju-log s3-credentials:16: Request PUT to https://10.138.71.161:9200/_snapshot/s3-repository with payload: {'type': 's3', 'settings': {'endpoint': 'http://localhost', 'protocol': 'http', 'bucket': 'error', 'base_path': '/', 'region': 'default'}} failed.(Attempts left: 2)
	Error: 500 Server Error: Internal Server Error for url: https://10.138.71.161:9200/_snapshot/s3-repository
unit-main-0: 01:58:52 ERROR unit.main/0.juju-log s3-credentials:16: Request PUT to https://10.138.71.161:9200/_snapshot/s3-repository with payload: {'type': 's3', 'settings': {'endpoint': 'http://localhost', 'protocol': 'http', 'bucket': 'error', 'base_path': '/', 'region': 'default'}} failed.(Attempts left: 1)
	Error: 500 Server Error: Internal Server Error for url: https://10.138.71.161:9200/_snapshot/s3-repository
unit-main-0: 01:58:53 ERROR unit.main/0.juju-log s3-credentials:16: Failed to setup backup service with state repository s3 is unreachable
unit-main-0: 01:59:02 ERROR unit.main/0.juju-log s3-credentials:16: Request PUT to https://10.138.71.161:9200/_snapshot/s3-repository with payload: {'type': 's3', 'settings': {'endpoint': 'http://localhost', 'protocol': 'http', 'bucket': 'error', 'base_path': '/', 'region': 'default'}} failed.(Attempts left: 5)
	Error: 500 Server Error: Internal Server Error for url: https://10.138.71.161:9200/_snapshot/s3-repository
unit-main-0: 01:59:03 ERROR unit.main/0.juju-log s3-credentials:16: Request PUT to https://10.138.71.161:9200/_snapshot/s3-repository with payload: {'type': 's3', 'settings': {'endpoint': 'http://localhost', 'protocol': 'http', 'bucket': 'error', 'base_path': '/', 'region': 'default'}} failed.(Attempts left: 4)
	Error: 500 Server Error: Internal Server Error for url: https://10.138.71.161:9200/_snapshot/s3-repository
unit-main-0: 01:59:05 ERROR unit.main/0.juju-log s3-credentials:16: Request PUT to https://10.138.71.161:9200/_snapshot/s3-repository with payload: {'type': 's3', 'settings': {'endpoint': 'http://localhost', 'protocol': 'http', 'bucket': 'error', 'base_path': '/', 'region': 'default'}} failed.(Attempts left: 3)
	Error: 500 Server Error: Internal Server Error for url: https://10.138.71.161:9200/_snapshot/s3-repository
unit-main-0: 01:59:06 ERROR unit.main/0.juju-log s3-credentials:16: Request PUT to https://10.138.71.161:9200/_snapshot/s3-repository with payload: {'type': 's3', 'settings': {'endpoint': 'http://localhost', 'protocol': 'http', 'bucket': 'error', 'base_path': '/', 'region': 'default'}} failed.(Attempts left: 2)
	Error: 500 Server Error: Internal Server Error for url: https://10.138.71.161:9200/_snapshot/s3-repository
unit-main-0: 01:59:07 ERROR unit.main/0.juju-log s3-credentials:16: Request PUT to https://10.138.71.161:9200/_snapshot/s3-repository with payload: {'type': 's3', 'settings': {'endpoint': 'http://localhost', 'protocol': 'http', 'bucket': 'error', 'base_path': '/', 'region': 'default'}} failed.(Attempts left: 1)
	Error: 500 Server Error: Internal Server Error for url: https://10.138.71.161:9200/_snapshot/s3-repository
unit-main-0: 01:59:09 ERROR unit.main/0.juju-log s3-credentials:16: Failed to setup backup service with state repository s3 is unreachable
unit-main-0: 01:59:19 ERROR unit.main/0.juju-log s3-credentials:16: Request PUT to https://10.138.71.161:9200/_snapshot/s3-repository with payload: {'type': 's3', 'settings': {'endpoint': 'http://localhost', 'protocol': 'http', 'bucket': 'error', 'base_path': '/', 'region': 'default'}} failed.(Attempts left: 5)
	Error: 500 Server Error: Internal Server Error for url: https://10.138.71.161:9200/_snapshot/s3-repository
unit-main-0: 01:59:21 ERROR unit.main/0.juju-log s3-credentials:16: Request PUT to https://10.138.71.161:9200/_snapshot/s3-repository with payload: {'type': 's3', 'settings': {'endpoint': 'http://localhost', 'protocol': 'http', 'bucket': 'error', 'base_path': '/', 'region': 'default'}} failed.(Attempts left: 4)
	Error: 500 Server Error: Internal Server Error for url: https://10.138.71.161:9200/_snapshot/s3-repository
unit-main-0: 01:59:22 ERROR unit.main/0.juju-log s3-credentials:16: Request PUT to https://10.138.71.161:9200/_snapshot/s3-repository with payload: {'type': 's3', 'settings': {'endpoint': 'http://localhost', 'protocol': 'http', 'bucket': 'error', 'base_path': '/', 'region': 'default'}} failed.(Attempts left: 3)
	Error: 500 Server Error: Internal Server Error for url: https://10.138.71.161:9200/_snapshot/s3-repository
unit-main-0: 01:59:23 ERROR unit.main/0.juju-log s3-credentials:16: Request PUT to https://10.138.71.161:9200/_snapshot/s3-repository with payload: {'type': 's3', 'settings': {'endpoint': 'http://localhost', 'protocol': 'http', 'bucket': 'error', 'base_path': '/', 'region': 'default'}} failed.(Attempts left: 2)
	Error: 500 Server Error: Internal Server Error for url: https://10.138.71.161:9200/_snapshot/s3-repository
unit-main-0: 01:59:25 ERROR unit.main/0.juju-log s3-credentials:16: Request PUT to https://10.138.71.161:9200/_snapshot/s3-repository with payload: {'type': 's3', 'settings': {'endpoint': 'http://localhost', 'protocol': 'http', 'bucket': 'error', 'base_path': '/', 'region': 'default'}} failed.(Attempts left: 1)
	Error: 500 Server Error: Internal Server Error for url: https://10.138.71.161:9200/_snapshot/s3-repository
unit-main-0: 01:59:26 ERROR unit.main/0.juju-log s3-credentials:16: Failed to setup backup service with state repository s3 is unreachable
unit-main-0: 01:59:38 ERROR unit.main/0.juju-log s3-credentials:16: Request PUT to https://10.138.71.161:9200/_snapshot/s3-repository with payload: {'type': 's3', 'settings': {'endpoint': 'http://10.145.253.243', 'protocol': 'http', 'bucket': 'testbucket', 'base_path': 'opensearch/4e5ca972-cbba-4215-a750-dfb0241da7e6', 'region': 'default'}} failed.(Attempts left: 5)
	Error: 500 Server Error: Internal Server Error for url: https://10.138.71.161:9200/_snapshot/s3-repository
unit-main-0: 01:59:40 ERROR unit.main/0.juju-log s3-credentials:16: Request PUT to https://10.138.71.161:9200/_snapshot/s3-repository with payload: {'type': 's3', 'settings': {'endpoint': 'http://10.145.253.243', 'protocol': 'http', 'bucket': 'testbucket', 'base_path': 'opensearch/4e5ca972-cbba-4215-a750-dfb0241da7e6', 'region': 'default'}} failed.(Attempts left: 4)
	Error: 500 Server Error: Internal Server Error for url: https://10.138.71.161:9200/_snapshot/s3-repository
unit-main-0: 01:59:41 ERROR unit.main/0.juju-log s3-credentials:16: Request PUT to https://10.138.71.161:9200/_snapshot/s3-repository with payload: {'type': 's3', 'settings': {'endpoint': 'http://10.145.253.243', 'protocol': 'http', 'bucket': 'testbucket', 'base_path': 'opensearch/4e5ca972-cbba-4215-a750-dfb0241da7e6', 'region': 'default'}} failed.(Attempts left: 3)
	Error: 500 Server Error: Internal Server Error for url: https://10.138.71.161:9200/_snapshot/s3-repository
unit-main-0: 01:59:42 ERROR unit.main/0.juju-log s3-credentials:16: Request PUT to https://10.138.71.161:9200/_snapshot/s3-repository with payload: {'type': 's3', 'settings': {'endpoint': 'http://10.145.253.243', 'protocol': 'http', 'bucket': 'testbucket', 'base_path': 'opensearch/4e5ca972-cbba-4215-a750-dfb0241da7e6', 'region': 'default'}} failed.(Attempts left: 2)
	Error: 500 Server Error: Internal Server Error for url: https://10.138.71.161:9200/_snapshot/s3-repository
unit-main-0: 01:59:44 ERROR unit.main/0.juju-log s3-credentials:16: Request PUT to https://10.138.71.161:9200/_snapshot/s3-repository with payload: {'type': 's3', 'settings': {'endpoint': 'http://10.145.253.243', 'protocol': 'http', 'bucket': 'testbucket', 'base_path': 'opensearch/4e5ca972-cbba-4215-a750-dfb0241da7e6', 'region': 'default'}} failed.(Attempts left: 1)
	Error: 500 Server Error: Internal Server Error for url: https://10.138.71.161:9200/_snapshot/s3-repository
unit-failover-1: 01:59:50 ERROR unit.failover/1.juju-log /snap/opensearch/current/usr/share/opensearch/bin/opensearch-keystore remove s3.client.default.access_key:
 Stderr: warning: no-jdk distributions that do not bundle a JDK are deprecated and will be removed in a future release
ERROR: Setting [s3.client.default.access_key] does not exist in the keystore.

 Stdout: 
unit-opensearch-0: 01:59:50 ERROR unit.opensearch/0.juju-log /snap/opensearch/current/usr/share/opensearch/bin/opensearch-keystore remove s3.client.default.access_key:
 Stderr: warning: no-jdk distributions that do not bundle a JDK are deprecated and will be removed in a future release
ERROR: Setting [s3.client.default.access_key] does not exist in the keystore.

 Stdout: 
unit-failover-1: 01:59:52 ERROR unit.failover/1.juju-log /snap/opensearch/current/usr/share/opensearch/bin/opensearch-keystore remove s3.client.default.access_key:
 Stderr: warning: no-jdk distributions that do not bundle a JDK are deprecated and will be removed in a future release
ERROR: Setting [s3.client.default.access_key] does not exist in the keystore.

 Stdout: 
unit-opensearch-0: 01:59:52 ERROR unit.opensearch/0.juju-log /snap/opensearch/current/usr/share/opensearch/bin/opensearch-keystore remove s3.client.default.access_key:
 Stderr: warning: no-jdk distributions that do not bundle a JDK are deprecated and will be removed in a future release
ERROR: Setting [s3.client.default.access_key] does not exist in the keystore.

 Stdout: 
unit-failover-1: 01:59:53 ERROR unit.failover/1.juju-log /snap/opensearch/current/usr/share/opensearch/bin/opensearch-keystore remove s3.client.default.access_key:
 Stderr: warning: no-jdk distributions that do not bundle a JDK are deprecated and will be removed in a future release
ERROR: Setting [s3.client.default.access_key] does not exist in the keystore.

 Stdout: 
unit-opensearch-0: 01:59:53 ERROR unit.opensearch/0.juju-log /snap/opensearch/current/usr/share/opensearch/bin/opensearch-keystore remove s3.client.default.access_key:
 Stderr: warning: no-jdk distributions that do not bundle a JDK are deprecated and will be removed in a future release
ERROR: Setting [s3.client.default.access_key] does not exist in the keystore.

 Stdout: 
unit-failover-1: 01:59:54 ERROR unit.failover/1.juju-log /snap/opensearch/current/usr/share/opensearch/bin/opensearch-keystore remove s3.client.default.secret_key:
 Stderr: warning: no-jdk distributions that do not bundle a JDK are deprecated and will be removed in a future release
ERROR: Setting [s3.client.default.secret_key] does not exist in the keystore.

 Stdout: 
unit-opensearch-0: 01:59:54 ERROR unit.opensearch/0.juju-log /snap/opensearch/current/usr/share/opensearch/bin/opensearch-keystore remove s3.client.default.secret_key:
 Stderr: warning: no-jdk distributions that do not bundle a JDK are deprecated and will be removed in a future release
ERROR: Setting [s3.client.default.secret_key] does not exist in the keystore.

 Stdout: 
unit-failover-1: 01:59:56 ERROR unit.failover/1.juju-log /snap/opensearch/current/usr/share/opensearch/bin/opensearch-keystore remove s3.client.default.secret_key:
 Stderr: warning: no-jdk distributions that do not bundle a JDK are deprecated and will be removed in a future release
ERROR: Setting [s3.client.default.secret_key] does not exist in the keystore.

 Stdout: 
unit-opensearch-0: 01:59:56 ERROR unit.opensearch/0.juju-log /snap/opensearch/current/usr/share/opensearch/bin/opensearch-keystore remove s3.client.default.secret_key:
 Stderr: warning: no-jdk distributions that do not bundle a JDK are deprecated and will be removed in a future release
ERROR: Setting [s3.client.default.secret_key] does not exist in the keystore.

 Stdout: 
unit-failover-1: 01:59:57 ERROR unit.failover/1.juju-log /snap/opensearch/current/usr/share/opensearch/bin/opensearch-keystore remove s3.client.default.secret_key:
 Stderr: warning: no-jdk distributions that do not bundle a JDK are deprecated and will be removed in a future release
ERROR: Setting [s3.client.default.secret_key] does not exist in the keystore.

 Stdout: 
unit-opensearch-0: 01:59:57 ERROR unit.opensearch/0.juju-log /snap/opensearch/current/usr/share/opensearch/bin/opensearch-keystore remove s3.client.default.secret_key:
 Stderr: warning: no-jdk distributions that do not bundle a JDK are deprecated and will be removed in a future release
ERROR: Setting [s3.client.default.secret_key] does not exist in the keystore.

 Stdout: 
unit-main-0: 02:02:49 ERROR unit.main/0.juju-log s3-credentials:19: Request PUT to https://10.138.71.161:9200/_snapshot/s3-repository with payload: {'type': 's3', 'settings': {'endpoint': 'http://10.145.253.243', 'protocol': 'http', 'bucket': 'testbucket', 'base_path': 'opensearch/4e5ca972-cbba-4215-a750-dfb0241da7e6', 'region': 'default'}} failed.(Attempts left: 5)
	Error: 500 Server Error: Internal Server Error for url: https://10.138.71.161:9200/_snapshot/s3-repository
unit-main-0: 02:02:51 ERROR unit.main/0.juju-log s3-credentials:19: Request PUT to https://10.138.71.161:9200/_snapshot/s3-repository with payload: {'type': 's3', 'settings': {'endpoint': 'http://10.145.253.243', 'protocol': 'http', 'bucket': 'testbucket', 'base_path': 'opensearch/4e5ca972-cbba-4215-a750-dfb0241da7e6', 'region': 'default'}} failed.(Attempts left: 4)
	Error: 500 Server Error: Internal Server Error for url: https://10.138.71.161:9200/_snapshot/s3-repository
unit-main-0: 02:02:52 ERROR unit.main/0.juju-log s3-credentials:19: Request PUT to https://10.138.71.161:9200/_snapshot/s3-repository with payload: {'type': 's3', 'settings': {'endpoint': 'http://10.145.253.243', 'protocol': 'http', 'bucket': 'testbucket', 'base_path': 'opensearch/4e5ca972-cbba-4215-a750-dfb0241da7e6', 'region': 'default'}} failed.(Attempts left: 3)
	Error: 500 Server Error: Internal Server Error for url: https://10.138.71.161:9200/_snapshot/s3-repository
unit-main-0: 02:02:53 ERROR unit.main/0.juju-log s3-credentials:19: Request PUT to https://10.138.71.161:9200/_snapshot/s3-repository with payload: {'type': 's3', 'settings': {'endpoint': 'http://10.145.253.243', 'protocol': 'http', 'bucket': 'testbucket', 'base_path': 'opensearch/4e5ca972-cbba-4215-a750-dfb0241da7e6', 'region': 'default'}} failed.(Attempts left: 2)
	Error: 500 Server Error: Internal Server Error for url: https://10.138.71.161:9200/_snapshot/s3-repository
unit-main-0: 02:02:55 ERROR unit.main/0.juju-log s3-credentials:19: Request PUT to https://10.138.71.161:9200/_snapshot/s3-repository with payload: {'type': 's3', 'settings': {'endpoint': 'http://10.145.253.243', 'protocol': 'http', 'bucket': 'testbucket', 'base_path': 'opensearch/4e5ca972-cbba-4215-a750-dfb0241da7e6', 'region': 'default'}} failed.(Attempts left: 1)
	Error: 500 Server Error: Internal Server Error for url: https://10.138.71.161:9200/_snapshot/s3-repository
unit-main-0: 02:03:06 ERROR unit.main/0.juju-log s3-credentials:19: Request PUT to https://10.138.71.161:9200/_snapshot/s3-repository with payload: {'type': 's3', 'settings': {'endpoint': 'http://10.145.253.243', 'protocol': 'http', 'bucket': 'testbucket', 'base_path': 'opensearch/4e5ca972-cbba-4215-a750-dfb0241da7e6', 'region': 'default'}} failed.(Attempts left: 5)
	Error: 500 Server Error: Internal Server Error for url: https://10.138.71.161:9200/_snapshot/s3-repository
unit-main-0: 02:03:07 ERROR unit.main/0.juju-log s3-credentials:19: Request PUT to https://10.138.71.161:9200/_snapshot/s3-repository with payload: {'type': 's3', 'settings': {'endpoint': 'http://10.145.253.243', 'protocol': 'http', 'bucket': 'testbucket', 'base_path': 'opensearch/4e5ca972-cbba-4215-a750-dfb0241da7e6', 'region': 'default'}} failed.(Attempts left: 4)
	Error: 500 Server Error: Internal Server Error for url: https://10.138.71.161:9200/_snapshot/s3-repository
unit-main-0: 02:03:09 ERROR unit.main/0.juju-log s3-credentials:19: Request PUT to https://10.138.71.161:9200/_snapshot/s3-repository with payload: {'type': 's3', 'settings': {'endpoint': 'http://10.145.253.243', 'protocol': 'http', 'bucket': 'testbucket', 'base_path': 'opensearch/4e5ca972-cbba-4215-a750-dfb0241da7e6', 'region': 'default'}} failed.(Attempts left: 3)
	Error: 500 Server Error: Internal Server Error for url: https://10.138.71.161:9200/_snapshot/s3-repository
unit-main-0: 02:03:10 ERROR unit.main/0.juju-log s3-credentials:19: Request PUT to https://10.138.71.161:9200/_snapshot/s3-repository with payload: {'type': 's3', 'settings': {'endpoint': 'http://10.145.253.243', 'protocol': 'http', 'bucket': 'testbucket', 'base_path': 'opensearch/4e5ca972-cbba-4215-a750-dfb0241da7e6', 'region': 'default'}} failed.(Attempts left: 2)
	Error: 500 Server Error: Internal Server Error for url: https://10.138.71.161:9200/_snapshot/s3-repository
unit-main-0: 02:03:11 ERROR unit.main/0.juju-log s3-credentials:19: Request PUT to https://10.138.71.161:9200/_snapshot/s3-repository with payload: {'type': 's3', 'settings': {'endpoint': 'http://10.145.253.243', 'protocol': 'http', 'bucket': 'testbucket', 'base_path': 'opensearch/4e5ca972-cbba-4215-a750-dfb0241da7e6', 'region': 'default'}} failed.(Attempts left: 1)
	Error: 500 Server Error: Internal Server Error for url: https://10.138.71.161:9200/_snapshot/s3-repository

[32mINFO    [0m pytest_operator.plugin:plugin.py:947 Forgetting model main...
[32mINFO    [0m integration.ha.test_backups:test_backups.py:173 Cleaning backups from cloud buckets